docs/deploy/gke.md

```diff
@@ -304,6 +305,98 @@ You can get the external IP address of your service using:
 kubectl get svc adk-agent -o=jsonpath='{.status.loadBalancer.ingress[0].ip}'
 ```
 
+### Option 2: Automated Deployment using `adk deploy gke`
+
+ADK provides a CLI command to streamline GKE deployment. This avoids the need to manually build images, write Kubernetes manifests, or push to Artifact Registry.
+
+#### Prerequisites
+
+Before you begin, ensure you have the following set up:
+
+1. **A running GKE cluster:** You need an active Kubernetes cluster on Google Cloud.
+
+2. **`gcloud` CLI:** The Google Cloud CLI must be installed, authenticated, and configured to use your target project. Run `gcloud auth login` and `gcloud config set project [YOUR_PROJECT_ID]`.
+
+3. **Required IAM Permissions:** The user or service account running the command needs, at a minimum, the following roles:
+
+   * **Kubernetes Engine Developer** (`roles/container.developer`): To interact with the GKE cluster.
+
+   * **Artifact Registry Writer** (`roles/artifactregistry.writer`): To push the agent's container image.
+
+4. **Docker:** The Docker daemon must be running on your local machine to build the container image.
```

Comment:
This doesn't appear correct. The changes in adk-python/pull/1607 use Cloud Build to build the container, which means we need to tell the user the Cloud Build API needs to be enabled in the project and they must have `roles/cloudbuild.builds.editor`.